Vic2ToHoI4/Source/OutHoi4/OutMod.cpp

```diff
@@ -54,7 +54,7 @@ void createOutputFolder(const std::string& outputName)
 
 void createModFiles(const std::string& outputName)
 {
-	LOG(LogLevel::Info) << "\tCreating .mod files";
+	Log(LogLevel::Info) << "\tCreating .mod files";
```

Comment:
Not relevant to this line but drop "user_dir=" line below. It's spamming the user folder for no reason as the game won't save saves inside anyway.